### PR TITLE
[lldb] Work around two failing tests by disabling swift-read-metadata-from-file-cache

### PR DESCRIPTION
**Pull request by kastiglione**

### CODE REVIEW

**Comment by kastiglione** [comment]
@swift-ci test

**Review by felipepiovezan — APPROVED** [review]

File: lldb/test/API/lang/swift/foundation_value_types/uuid/main.swift [hunk 1]
```diff
@@ -12,6 +12,7 @@
 import Foundation
 
 func main() {
+  //% self.runCmd("settings set target.experimental.swift-read-metadata-from-file-cache false")
```
Comment (augusto2112) [inline lldb/test/API/lang/swift/foundation_value_types/uuid/main.swift:15]: Why is this one commented out?

Comment (felipepiovezan) [reply lldb/test/API/lang/swift/foundation_value_types/uuid/main.swift:15]: that's the syntax for those "inlined" tests

Comment (augusto2112) [reply lldb/test/API/lang/swift/foundation_value_types/uuid/main.swift:15]: Oh right

**Comment by augusto2112** [comment]
Can you hold on merging this? I have a fix ready

**Comment by augusto2112** [comment]
Here https://github.com/swiftlang/llvm-project/pull/9524

**Pull request closed without merging**